test/common/router/router_test.cc

```diff
@@ -6150,7 +6215,7 @@ TEST_F(RouterTest, NotSetDynamicMaxStreamDurationIfZero) {
         return nullptr;
       }));
 
-  // The timer will not be created.
+  // The timer will not be created.over
```

Comment:
This looks like it was changed on accident. Revert please.

Reply:
Done. Thanks for catching it.